lua/lspconfig.lua

```diff
@@ -80,8 +94,8 @@ local minimum_neovim_version = '0.9'
 if vim.fn.has('nvim-' .. minimum_neovim_version) == 0 then
   local msg = string.format(
     'nvim-lspconfig requires Nvim version %s, but you are running: %s',
-    vim.version.parse(minimum_neovim_version),
-    vim.version()
+    minimum_neovim_version,
+    version_string(vim.version())
```

Comment:
`vim.version()` has been around [longer](https://github.com/neovim/neovim/commit/61437c20b5936de1f833d155b7287d9151e763d8) than `.parse()`, so that's why it's ok to use it. But to be extra safe, we could do:

```suggestion
    version_string(vim.version and vim.version() or 'older than v0.5.0')
```

Reply:
I like it, I'll add this with some fixes